pkg/bytecode/vm_test.go

```diff
@@ -11,11 +11,8 @@ import (
 func TestVMGlobals(t *testing.T) {
 	tests := []testCase{
 		{
-			name: "global assignment",
-			input: `
-			x := 1
-			x = x
-			`,
+			name:              "global assignment",
+			input:             "x := 1",
 			expectedStackTop:  1,
 			expectedConstants: []any{1},
 			expectedInstructions: []Instructions{
```

Comment:
This value still has the trailing `GetGlobal`/`SetGlobal` which should be removed here and managed in the test itself.